### ISSUE TITLE
Error when validating during training

### ISSUE THREAD

**Issue by dandys2** [fizyr/keras-retinanet#1185]
I get following error when I train with validation data using 

retinanet-train --gpu=0 --snapshot-path snapshots/img_size300_100steps_batch8_freezebckb --random-transform --freeze-backbone --image-min-side=300 --image-max-side=500 --compute-val-loss  --epochs=10 --steps=100 --batch-size=8 --compute-val-loss csv train_data.csv classmap.csv --val-annotations valid_data.csv

running this, after first epoch I get:

![image](https://user-images.githubusercontent.com/25956904/68995970-2d838f00-0894-11ea-93d1-ae2808a3705e.png)

When running without validation data, everything works fine. I even checked my validation file but it's been created the same way as my train file, so that should be ok. My class indices start at 0, that should be fine too.
Any ideas why the stated error shows up?

Thanks for any help.

**Comment by dandys2** [comment]
Ok, meanwhile I fixed it, but I believe it's your bug in callbacks/eval.py in function on_epoch_end

  average_precisions, **inference_time** = evaluate(
            self.generator,
            self.model,
            iou_threshold=self.iou_threshold,
            score_threshold=self.score_threshold,
            max_detections=self.max_detections,
            save_path=self.save_path
        )

after adding the second return parameter **inference_time** (same as in bin/evaluate.py), it solved the issue.

**Comment by hgaiser** [comment]
Thanks for the feedback! It is indeed an oversight on our side, there is a PR to fix it, which will be merged soon :)
